.github/workflows/build-test-docker.yml

```diff
@@ -0,0 +1,159 @@
+name: build-test-docker
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
```

Comment:
I think we should add a check like https://github.com/FRRouting/frr/blob/master/.github/workflows/docker-daily-master.yml#L13 to avoid running these on the forks.

Reply:
Rather than use the `if` conditional, i've limited it using `on:` to pushes to master and stable branches (merges) and pull requests. This should avoid running on most forks (although it's fine if it does as it won't deplete execution credits for the user), and won't send email to users about the job being skipped as the conflicts.yml job currently does.

Reply:
I think it won't work unless we do `pull_request_target` due to permissions, but let's see.